src/main/java/org/gradle/playframework/tools/internal/javascript/GoogleClosureCompiler.java

```diff
@@ -49,7 +49,7 @@ public WorkResult execute(JavaScriptCompileSpec spec) {
         if (allErrors.isEmpty()) {
             return WorkResults.didWork(true);
         } else {
-            throw new SourceTransformationException(String.format("Minification failed with the following errors:\n\t%s", String.join( "\n\t", allErrors)), null);
+            throw new GradleException(String.format("Minification failed with the following errors:\n\t%s", String.join( "\n\t", allErrors)), null);
```

Comment:
why pass `null` as the last argument here?

Reply:
I just replaced the exception type and left it as it was. Probably the other exception missed the single argument constructor. You are right that this is just noise. Removed. Good catch!